main/integration-tests/support/resources/aws-accounts/aws-accounts.js

```diff
@@ -69,6 +70,18 @@ class AwsAccounts extends CollectionResource {
     return awsAccount;
   }
 
```

Comment:
Do we need test for update aws account as well?

Reply:
hmmmmm...it wouldn't hurt haha